mpd/README.md

```diff
@@ -3,296 +3,506 @@
 ## Requirements
 
 You only need one or more Music Player Daemons (MPD).
+For more information about MPD and it's commands see http://www.musicpd.org/doc/protocol/command_reference.html
 
 ## Configuration
 
-### plugin.conf (deprecated) / plugin.yaml
-
-```
-[mpd]
-    class_name = MPD
-    class_path = plugins.mpd
-```
+### plugin.yaml
 
 ```yaml
 mpd:
     class_name: MPD
     class_path: plugins.mpd
+    instance: bathroom
+    host: 192.168.177.164
+    host: 192.168.177.164
```

Comment:
Duplicate `host` attribute necessary?

In case it is always an IP address the `plugin.yml` can be changed to declare this field as a `ip` field instead of `str`. But I guess it could also be just a hostname.

Reply:
Yes, I just testet using the hostname instead of the IP in the plugin.yaml. Works as expected with a hostname too.

Reply:
OK, thanks for testing this. But I meant to change the `type` of the `host` parameter in the `plugin.yml` from `str` (current version) to `ip`. But I guess this is not possible because `ip` type is matching against a valid IP string. So hostnames are not valid for `ip` type. @msinn can tell us more about the parameter validation.

For the moment it's OK as it is currently.

Reply:
At the moment there are two datatypes that can be used for hostnames:
- ip - a string, representing a hostname or an ipv4-address
- ipv4 - a string, representing strictly an ipv4-address

Metadata is only documented in the development documentation of the (not yet released) v1.4: https://www.smarthomeng.de/dev/developer/development_plugin/plugin_metadata.html